cli/src/test/deploy.spec.ts

```diff
@@ -153,3 +153,15 @@ test('deploy fails with --app-name and --new-app', async (t) => {
   const result = await t.context.shell.run(`${t.context.run} deploy --app-name snarky-ninja --new-app`, 'utf-8');
   t.snapshot(result);
 });
+
+test('deploy with json format returns only app json', async (t) => {
+  const app = createApp({ id: 'abc' });
+  td.when(t.context.lycanFake.listApps(anything)).thenResolve([]);
+  td.when(t.context.uploadFake()).thenReturn({ ok: true, digest: 'abcabc' });
+  td.when(t.context.lycanFake.deployInitial(anything, 'default', 'abcabc', [])).thenResolve(app);
+  t.context.projectConfig = JSON.stringify({ accessToken: 'setec-astronomy' });
+  await writeFile(t.context.configPath, t.context.projectConfig);
+
+  const jsonResult = await t.context.shell.run(`${t.context.run} deploy --format json`, 'utf-8');
+  t.snapshot(JSON.parse(jsonResult.out));
```

Comment:
Can you snapshot `t.snapshot(result)` - so that both `result.out` and `result.err` have snapshots?